src/pages/questions/index.tsx

```diff
@@ -175,6 +191,17 @@ function Questions() {
             Peer<span className="text-[var(--txt-1)]">Prep</span>
           </h1>
           <div className="text-[var(--txt-3)] w-full flex-1 flex flex-col rounded overflow-hidden">
+            <div className="flex-1 flex gap-2 mb-4">
+              <StyledButton
```

Comment:
There's a select page div that we cld use for this in `components/code`, but will have to export that component

Reply:
Ok